src/main/groovy/life/qbic/datamodel/dtos/general/Person.groovy

```diff
@@ -46,13 +41,11 @@ abstract class Person {
    */
   final List<Affiliation> affiliations
 
-  Person(String personType,
-         String firstName,
+  Person(String firstName,
```

Comment:
This change breaks all implementations of the Customer class. 

It would be preferable to mark it as @deprecated and introduce a Builder without the type instead